installer/ansible/group_vars/srm-toolchain.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 The SODA Authors.
```

Comment:
Any reason for 2020 ? (For all occurrences)

Reply:
Done